Xamarin.Forms.Platform.Android/Renderers/ShellContentFragment.cs

```diff
@@ -82,7 +82,10 @@ public override AndroidAnimation OnCreateAnimation(int transit, bool enter, int
 			}
 
 			if (result == null)
+			{
+				AnimationFinished?.Invoke(this, EventArgs.Empty);
```

Comment:
Basically if the user creates a custom renderer that doesn't actually add an animation then the following code

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/Renderers/ShellItemRendererBase.cs#L254

will never execute and the "async push" operation will never complete

This invokes the AnimationFinished event so the calling code can continue on and complete the animation Task